en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/viewing-api-statistics.md

```diff
@@ -498,3 +487,25 @@ The status of the APIs (all API versions) represented in a tabular view.
 </html>
 
 The availability of APIs statistics is directly related to the [Availability of APIs (health monitoring)](../../../../learn/analytics/managing-alerts-with-real-time-analytics/alert-types/#availability-of-apis-api-health-monitoring) alert type.
+
+## Date Range Picker
+
+This can be used to configure the **time period** and **granularity mode** of the data should be visualized or reported in the widgets listed above. Here the aggregated data to the selected granularity will be visualized or reported in widgets.
```

Comment:
```suggestion
This can be used to configure the **time period** and **granularity mode** of the data which should be visualized in the widgets listed above.
```

Reply:
Resolved with bea00a769294fb86757ff2bc3fb05557bffa47f2